apps/frontend/src/components/proposal/ProposalInviteNotification.tsx

```diff
@@ -0,0 +1,166 @@
+        {
+          variant: 'error',
+        }
+      );
+    }
+  };
+
+  return (
+    <>
+      <Box
+        data-testid="proposal-invite-notification"
+        sx={{
+          backgroundColor: (theme) => alpha(theme.palette.info.main, 0.12),
+          border: (theme) => `1px solid ${alpha(theme.palette.info.main, 0.5)}`,
+          color: 'info.main',
+          borderRadius: 1,
+          padding: 2,
+          marginBottom: 2,
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          width: '100%',
+        }}
+      >
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+          <PersonAddIcon />
+          <Typography variant="body1" color="textSecondary">
+            You have {inviteCount} outstanding invitation
+            {inviteCount > 1 ? 's' : ''} to join proposal
+            {inviteCount > 1 ? 's' : ''}.
+          </Typography>
+        </Box>
+        <Button
+          data-testid="view-invitations-btn"
+          variant="contained"
+          size="small"
+          onClick={() => setIsDialogOpen(true)}
+          disabled={loading}
+          sx={{ marginLeft: 2 }}
+        >
+          {loading ? 'Loading...' : 'View Invitations'}
+        </Button>
+      </Box>
+
+      <Dialog
+        data-testid="proposal-invite-dialog"
+        open={isDialogOpen}
+        onClose={() => setIsDialogOpen(false)}
+        maxWidth="md"
+        fullWidth
+      >
+        <DialogTitle>
+          <Typography variant="h6">Proposal Invitations</Typography>
+        </DialogTitle>
+        <DialogContent>
+          {proposalInvites.length === 0 ? (
+            <Typography variant="body1" color="textSecondary">
+              No pending invitations found.
+            </Typography>
+          ) : (
+            proposalInvites.map((invite) => (
+              <>
+                {invite.proposal && (
+                  <Box
+                    key={invite.id}
+                    sx={{
+                      backgroundColor: (theme) =>
+                        alpha(theme.palette.info.main, 0.12),
+                      border: (theme) =>
+                        `1px solid ${alpha(theme.palette.info.main, 0.5)}`,
+                      color: 'info.main',
+                      borderRadius: 1,
+                      padding: 2,
+                      marginBottom: 2,
+                      display: 'flex',
+                      alignItems: 'center',
+                      justifyContent: 'space-between',
+                      width: '100%',
+                    }}
+                  >
+                    <div>
+                      <Typography variant="subtitle1" fontWeight="bold">
+                        {invite.proposal.title || 'No Title'}
+                      </Typography>
+                      <Typography variant="body1" color="textSecondary">
+                        Principal Investigator: {invite.proposal.proposerName}
```

Comment:
If inviting others is not restricted to the proposer, is it useful to show who has sent the invite?